README.md

```diff
@@ -61,25 +61,39 @@ Currently, it's also possible to export and import styles in GUI. Styles importe
+| `label-outline-color`      | Text outline color of the label.                                                                                          | String                                                     | `#111111`                |
+| `label-outline-width`      | Text outline width of the label.                                                                                          | Float                                                      | `1.0`                    |
+| `label-font`               | The font used to draw the label (using the same syntax as the CSS 'font' property).                                       | String                                                     | `24px Helvetica`         |
+| `label-background-color`   | Background color of the label.                                                                                            | String                                                     | `#000000`                |
+| `label-background-padding` | Background padding in pixels.                                                                                             | Pair of Integers.                                          | `[7, 5]`                 |
+| `label-horizontal-origin`  | Determines if the label is drawn to "LEFT", "CENTER", or "RIGHT" of its anchor position.                                  | String                                                     | `LEFT`                   |
+| `label-vertical-origin`    | Determines if the label is to "ABOVE", "BELOW", at "CENTER" or at "BASELINE" of its anchor position.                      | String                                                     | `BASELINE`               |
+| `label-text-expression`    | A Simfil expression to evaluate on the feature the label belongs to.                                                      | String                                                     | `**.speedLimitKmh`       |
+| `label-text`               | A placeholder in case the simfil expression either isn't necessary or won't produce a result.                             | String                                                     | `No speed limit`         |
+| `label-style`              | Describes how to draw a label using "FILL", "OUTLINE" or "FILL_AND_OUTLINE".                                              | String                                                     | `FILL`                   |
+| `label-scale`              | The uniform scale that is multiplied with the label's size in pixels.                                                     | Float                                                      | `1.0`                    |
+| `label-pixel-offset`       | The offset in screen space from the origin of this label (the screen space origin is the top, left corner of the canvas). | Pair of Floats (optional).                                 | `[5.0, 30.0]`            |
```

Comment:
Nice docs :) But the "optional" designation in the type description is not needed. To mark a style field as optional, the name should be annotated with a bold star (__*__). However, since 99% of style rules are optional, maybe we should switch the meaning of the star to "mandatory". Which is really only the case for the `geometry` field.